src/it/scala/io/github/spark_redshift_community/spark/redshift/RedshiftWriteSuite.scala

```diff
@@ -167,3 +167,51 @@ class CSVGZIPRedshiftWriteSuite extends IntegrationSuiteBase {
     }
   }
 }
+
+class CSVGZIPCaseSensitiveRedshiftWriteSuite extends BaseRedshiftWriteSuite {
+  override protected def write(df: DataFrame): DataFrameWriter[Row] =
+    super.write(df).option("tempformat", "CSV GZIP").option("enable_case_sensitive_identifier", "true")
+
+  test("roundtrip save and load case sensitive") {
```

Comment:
@jsleight I've added two test here. I've tried to execute them using `sbt test` but it seems only tests in `src/test` get executed. How can I execute also these classes?

Reply:
When I execute using `sbt it:test` I get this error:
`Invalid operation: relation "PUBLIC.reading_decimal_19_0_8788118925352793780" does not exist;`

Reply:
That is weird.  @88manpreet do you remember seeing anything like this error before?